test/regression/2016-12-14-alloc-alignment.c

```diff
@@ -0,0 +1,21 @@
+// RUN: %llvmgcc %s -emit-llvm -g -O0 -c -o %t.bc
+// RUN: rm -rf %t.klee-out
+// RUN: %klee --output-dir=%t.klee-out --exit-on-error %t.bc
+#include <assert.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <string.h>
+
+// Global should be aligned on a 128-byte boundary
+int foo __attribute__((align(128)));
```

Comment:
Good catch. I didn't spot this because I forgot to pass `-Wall` to the compiler.